spec/index.html

```diff
@@ -250,7 +250,7 @@ <h2 id="file-structures"><a class="toc-anchor" href="#file-structures">§</a> Fi
 <img src="../docs/diagrams/file-topology.svg" style="display: block; margin: 0 auto; padding: 2em 0; width: 100%; max-width: 620px;" />
 <h3 id="anchor-file"><a class="toc-anchor" href="#anchor-file">§</a> Anchor File</h3>
 <p>Anchor Files contain <a href="#create">Create</a>, <a href="#recover">Recover</a>, and <a href="#deactivate">Deactivate</a> operation values, as well as a CAS URI for the related Sidetree Map file (detailed below). As the name suggests, Anchor Files are anchored to the target ledger system via embedding a CAS URI in the ledger’s transactional history.</p>
-<div id="example-313" class="notice example"><a class="notice-link" href="#example-313">EXAMPLE</a><pre class="language-json"><code class="language-json"><span class="token punctuation">{</span>
+<div id="example-1" class="notice example"><a class="notice-link" href="#example-1">EXAMPLE</a><pre class="language-json"><code class="language-json"><span class="token punctuation">{</span>
```

Comment:
committing spec up files is not great for review ability... CC @csuwildcat .... having spent a bunch of time with respec, I find this tool more trouble than its worth.

Reply:
@OR13 Should I be committing the file?

Reply:
Fixed.